app/src/main/java/com/mapbox/services/android/navigation/testapp/example/ui/ExampleActivity.kt

```diff
@@ -261,4 +261,11 @@ class ExampleActivity : AppCompatActivity(), ExampleView {
     val granted = PermissionsManager.areLocationPermissionsGranted(this)
     presenter.onPermissionResult(granted)
   }
+
+  override fun onBackPressed() {
+    var exitActivity = presenter.onBackPressed()
+    if ( exitActivity ){
```

Comment:
NIT - I know we don't have checkstyle rules for Kotlin set up yet. Could we fix this wrong formatting manually for now?